config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: controller
+  newTag: latest
```

Comment:
you could also do that the way CAPI does : https://github.com/kubernetes-sigs/cluster-api/blob/master/config/manager/manager_image_patch.yaml